inc/Integrations/Airtable/AirtableIntegration.php

```diff
@@ -16,8 +16,13 @@ public static function register_blocks(): void {
 			REMOTE_DATA_BLOCKS_AIRTABLE_SERVICE
 		);
 
-		foreach ( $data_source_configs as $config ) {
+		foreach ( $data_source_configs as $config ) {       
 			$data_source = AirtableDataSource::from_array( $config );
+
+			if ( false === $config['service_config']['enable_blocks'] ) {
+				continue;
+			}
+			
 			self::register_blocks_for_airtable_data_source( $data_source );
 			self::register_loop_blocks_for_airtable_data_source( $data_source );
 		}
```

Comment:
I personally prefer this style, but it's just personal preference. Take it or leave it.
```suggestion

			if ( false !== $config['service_config']['enable_blocks'] ) {
				self::register_blocks_for_airtable_data_source( $data_source );
				self::register_loop_blocks_for_airtable_data_source( $data_source );
			}
		}
```

Reply:
FWIW, I love early returns / breaks! As I read the code, it helps me "drop" logical gates that I don't need to think about anymore.

```
if ( condition_a ) {
  return;
}

if ( condition_b ) {
  return;
}

// Yay, all code from here out is guaranteed not to have to
// deal with `condition_a` or `condition_b`!!
```

Reply:
I'd lean towards early returns/breaks as well. It makes the rest of the code so much easier to read and work with, as the problematic conditions have been dealt with already

Reply:
Thanks @ingeniumed for chiming in so I didn't have to chose a side 😄 majority wins!

Reply:
I love simple early returns also, but for some reason a `continue` in a loop just feels more complex.

But I'm fine with it.